src/bindings/bnd_light.cpp

```diff
@@ -17,6 +17,19 @@ void BND_Light::SetTrackedPointer(ON_Light* light, const ON_ModelComponentRefere
   BND_GeometryBase::SetTrackedPointer(light, compref);
 }
 
+BND_TUPLE BND_Light:: GetSpotLightRadii() const
```

Comment:
This is legitimate code, but stylistically I would get rid of the space
`BND_TUPLE BND_Light::GetSpotLightRadii() const`